dev/ant_build.js/build.gradle

```diff
@@ -0,0 +1,139 @@
+/*******************************************************************************
+ * Copyright (c) 2020 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+
+import groovy.json.JsonSlurper
+
+configurations {
+    libs
+    libs1
+    libs2
+    libs3
+    libs4
+    libs5
+    libs6
+    libs7
+    libs8
+    libs9
+    libs10
+}
+
+dependencies {
+    libs "com.fasterxml.jackson.core:jackson-annotations:2.2.3"
+    libs "com.fasterxml.jackson.core:jackson-core:2.2.3"
```

Comment:
I don't like having all these version numbers listed here. Many of these are old versions and will be lacking various fixes. jackson for example was recently updated to pick up an important fix; https://github.com/OpenLiberty/open-liberty/pull/14168

Reply:
Also, this project isn't really ant based is it? Shouldn't it be given a different name?

Reply:
@k8vance88 @fritze2 any thoughts on this?